### PR TITLE
task/remove-undo-feature

### PR DESCRIPTION
**Pull request by michael-jaia**

### CODE REVIEW

File: src/web/command_control/client/components/CommandControl.tsx [hunk 1]
```diff
@@ -1264,6 +1241,17 @@ export default class CommandControl extends React.Component {
...
+	/**
+	 * Delete a run from the mission
+	 * 
+	 * @param {string} runId determines which run to delete
+	 * @param {string} disableMessage see @notes/refactor
+	 * @returns {void}
+	 * 
+	 * @notes
+	 * /refactor >> remove disable message as a parameter...this should be handled by an on click
+	 * 				handler for the clear run button
```
Comment (ferrom-jaia) [inline src/web/command_control/client/components/CommandControl.tsx]: Why is there a big space here?

File: src/web/command_control/client/components/CommandControl.tsx [hunk 2]
```diff
@@ -1640,7 +1631,7 @@ export default class CommandControl extends React.Component {
 	 */
```
Comment (ferrom-jaia) [inline src/web/command_control/client/components/CommandControl.tsx:1631]: remove - in function comment

**Review by ferrom-jaia — CHANGES_REQUESTED** [review]

File: src/web/command_control/client/components/CommandControl.tsx (hunk 1 above)
Comment (michael-jaia) [reply src/web/command_control/client/components/CommandControl.tsx]: It's that weird vs code to github conversion...here is how it looks in vs code...should I remove the space?
Screenshot from 2023-11-16 13-46-21

**Review by michael-jaia — COMMENTED** [review]
Responded to comments

**Review by ferrom-jaia — APPROVED** [review]
Looks great! Tested in sim.

**Pull request merged**